R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -95,6 +95,10 @@ test_that("spark.glm summary", {
   expect_equal(stats$df.residual, rStats$df.residual)
   expect_equal(stats$aic, rStats$aic)
 
+  out <- capture.output(print(stats))
+  expect_match(out[2], "Deviance Residuals:")
+  expect_true(any(grepl("AIC: 59.22", out)))
```

Comment:
Are we trying to test `print.summary.GeneralizedLinearRegressionModel` function here?